### PR TITLE
Small fixes for gpu-next and wayland

### PR DESCRIPTION
**Pull request by cyanreg**
Something that doesn't really matter outside of my own experiment for gpu-next.

And a small fix for Wayland. It shouldn't be possible to have 2 presentation feedback events in-fly, so it should be safe to always zero it out.

### CODE REVIEW

**Review by Dudemanguy — APPROVED** [review]
LGTM. I was too stupid to understand why the NULL was needed but it's quite obvious looking at it again.

Can't imagine the vo_gpu_next change has any deep implications, but I'll let @haasn ack that.

File: video/out/vo_gpu_next.c [hunk 1]
```diff
@@ -1238,7 +1238,7 @@ static int control(struct vo *vo, uint32_t request, void *data)
     case VOCTRL_PAUSE:
         if (p->is_interpolated)
             vo->want_redraw = true;
-        return VO_TRUE;
+        break;
```
Comment (haasn) [inline video/out/vo_gpu_next.c:1241]: Can you add at least *some* commit message explaining this change? What bug does it fix? Why here and not in `vo_gpu`?

**Comment by Dudemanguy** [comment]
Cherry-picked and pushed the second commit.

**Comment by sfan5** [comment]
@Dudemanguy can we just consistently get rid of dangling pointers in the wayland code?

**Comment by Dudemanguy** [comment]
The presentation stuff should have been the only ones.

File: video/out/vo_gpu_next.c (hunk 1 above)
Comment (haasn) [reply video/out/vo_gpu_next.c:1241]: But `vo_gpu` also swallows up the `VOCTRL_PAUSE` in the exact same place:

https://github.com/mpv-player/mpv/blob/master/video/out/vo_gpu.c#L216

**Pull request closed without merging**